eng/versioning/version_client.txt

```diff
@@ -53,3 +53,4 @@ com.azure:perf-test-core;1.0.0-beta.1;1.0.0-beta.1
 # beta_<groupId>:<artifactId>;dependency-version
 # note: Released beta versions will not be manipulated with the automatic PR creation code.
 unreleased_com.azure:azure-core-amqp;1.1.0-beta.1
+beta_com.azure:azure-identity;1.1.0-beta.3
```

Comment:
I think this should be removed since nothing is using it yet, don't want to add these eagerly.

Reply:
From the description of this feature.
This is needed for Identity as we don't want post release automated PRs to change dep version of Identity with beta versions.
But, its best to have @JimSuplizio  review this, so will add it via separate PR.

Reply:
Removed for now.